pom.xml

```diff
@@ -20,20 +20,30 @@
         <jacoco.version>0.8.5</jacoco.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.keycloak.bom</groupId>
+                <artifactId>keycloak-spi-bom</artifactId>
+                <version>${keycloak.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
```

Comment:
Is the idea here to refactor so we only have one place where we need to update the keycloak version in the future?

Reply:
yep, that's correct. This allows us to pin the version of any dependency this project ends up using that is part of the `keycloak-spi-bom` artifact to keep everything on the same page (like seen below)